### PR TITLE
[vNext] Prepare v3 pre-release 1

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Making a draft until we get deployment updated correctly for release

## Release Notes

Milestone: vNext

Changelog: n/a

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

File: .github/release.config.js [hunk 1]
```diff
@@ -15,10 +15,10 @@ module.exports = {
             channel: "zowe-v1-lts",
             level: "patch",
         },
-        // {
-        //     name: "next",
-        //     prerelease: true,
-        // }
+        {
+            name: "next",
+            prerelease: true,
```
Comment (zFernand0) [inline .github/release.config.js:20]: No need to change this, but wondering if we should be explicit with our `channel`s?

File: lerna.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.0.0-SNAPSHOT",
+  "version": "3.0.0-next.202308311221",
```
Comment (zFernand0) [inline lerna.json:2]: I don't think we need to specify a pre-release string here.
I don't believe it will hurt to keep it though.

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 
Thanks for cleaning up the other changelog files 😋

**Review by rudyflores — APPROVED** [review]

**Review by traeok — APPROVED** [review]

**Comment by zFernand0** [comment]
before merging...
@JillieBeanSim 
Can we update the engine version number on each of the VSCE packages (ZE and FTP) to at least 1.63 to prevent the error below from happening in this repo.
- https://github.com/zowe/zowe-cli-secrets-for-kubernetes/actions/runs/6041214803/job/16509428694

**Review by zFernand0 — DISMISSED** [review]
see comment above 😋

**Review by rudyflores — APPROVED** [review]

**Review by awharn — APPROVED** [review]

**Review by traeok — APPROVED** [review]
LGTM, thanks @JillieBeanSim

File: .github/release.config.js (hunk 1 above)
Comment (t1m0thyj) [reply .github/release.config.js:20]: Sorry for the delay 😢 The `channel` defaults to the `name`, but its probably good practice to be explicit like we are for the other branches 😋 We could add `channel: "next"` like the following:
```javascript
    {
        name: "next",
        channel: "next",
        prerelease: true
    }
```

File: lerna.json (hunk 2 above)
Comment (t1m0thyj) [reply lerna.json:2]: +1 since the version should get automatically updated by our deploy workflow to be `3.0.0-next.<timestamp>` at time of publish

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**